flink-table/flink-table-api-scala-bridge/src/main/scala/org/apache/flink/table/api/bridge/scala/internal/StreamTableEnvironmentImpl.scala

```diff
@@ -292,6 +294,12 @@ class StreamTableEnvironmentImpl (
 
 object StreamTableEnvironmentImpl {
 
+  def create(
+              executionEnvironment: StreamExecutionEnvironment,
+              settings: EnvironmentSettings): StreamTableEnvironmentImpl =
+    create(executionEnvironment, settings, TableConfig.getDefault)
+
+  @VisibleForTesting
```

Comment:
It's only used by `OverAggregateHarnessTest` because of the "weird" testing with a custom `TestTableConfig` class to explicitly set `maxIdleStateRetentionTime`, overriding the default calculation on the original `TableConfig`. Haven't thought of a workaround to avoid this, any suggestions?

Reply:
Let's rework the test then. I don't see a reason why we should make our internal methods complicated just for a wrongly implemented test.